lib/blemulator/blemulator_core.dart

```diff
@@ -0,0 +1,12 @@
+part of blemulator;
+
+class Blemulator {
+
+  BlemulatorBridge _bridge;
+
+  Blemulator() {
+    _bridge = BlemulatorBridge();
```

Comment:
Why don't we just instantiate this bridge in it's declaration line? We don't inject it (maybe we should?) so currently there is no need to have this default constructor.